tests/jsx-significant-space/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -101,24 +92,20 @@ before_break2 = (
     <span
       barbarbarbarbarbarbarbarbarbarbarbarbarbarbarbarbarbarbarbarbarbarbarbar
     />
-    {" "}
-    foo
+    {" "}foo
   </span>
 );
 
 after_break = (
   <span>
     foo
-    {" "}
-    <span barbarbarbarbarbarbarbarbarbarbarbarbarbarbarbarbarbarbarbarbarbar />
+    {" "}<span
```

Comment:
I'm interested in your thoughts on this. Based on this issue (https://github.com/prettier/prettier/issues/1582) I went with having `{" "}` at the start of lines. But I don't have strong feeling either way!